docs: update main readme (#350)
kyma-project/cloud-manager: README.md

# Cloud Manager

## Status

[![REUSE status](https://api.reuse.software/badge/github.com/kyma-project/cloud-manager)](https://api.reuse.software/info/github.com/kyma-project/cloud-manager)

![GitHub tag checks state](https://img.shields.io/github/checks-status/kyma-project/cloud-manager/main?label=cloud-manager&link=https%3A%2F%2Fgithub.com%2Fkyma-project%2Fcloud-manager%2Fcommits%2Fmain)

## Overview

> [Cloud Manager](docs/user#what-is-cloud-manager) is a central component that manages access to additional hyperscaler resources from the Kyma Runtime cluster. Its responsibility is to bring hyperscaler products/resources into the Kyma cluster in a secure way.

Reviewer: ```suggestion
[Cloud Manager](docs/user#what-is-cloud-manager) is a central component that manages access to additional hyperscaler resources from the Kyma cluster. It brings hyperscaler products and resources into the Kyma cluster in a secure way.
```
